src/config/dynamic-theme-fixes.config

```
@@ -14811,6 +14811,16 @@ IGNORE INLINE STYLE
 
 ================================
 
+wanikani.com
+
+CSS
+#question-type.meaning {
+    color: rgb(214, 209, 201) !important;
```

What about changing this to the dynamic variable?
```suggestion
    color: var(--darkreader-neutral-color) !important;
```

---

Good idea. Just tested and this looks solid to me.